crates/networking/p2p/discv4/server.rs

```diff
@@ -97,7 +99,8 @@ impl DiscoveryServer {
     ) -> Result<(), DiscoveryServerError> {
         info!("Starting Discovery Server");
 
-        let local_node_record = NodeRecord::from_node(&local_node, 1, &signer)
+        let fork_id = storage.get_fork_id().await.ok();
```

Comment:
This won't work correctly for nodes that are syncing. At startup they'll use the fork-id of the genesis block (latest block is 0), and will never update this once synced.

I think we need to have some periodic message that updates this once per X mins or something like that.

Reply:
Yeah. That's a good idea.